vm/src/pyobject.rs

```diff
@@ -705,6 +721,10 @@ impl<T: PyValue> PyRef<T> {
     pub fn typ(&self) -> PyClassRef {
         self.obj.class()
     }
+
+    pub fn into_generic_pyobj(self) -> Rc<PyObject<T>> {
```

Comment:
I don't have a good substitution but `generic_pyobj` gives me upcasting feeling to me.

Reply:
Ah, I see what you mean; "generic" as sort of general/untyped. Maybe `into_typed_pyobj()`?

Reply:
that sounds more clear to me